superset-frontend/src/theme/index.ts

```diff
@@ -31,7 +31,7 @@ const themes = {
 
 const baseConfig: ThemeConfig = {
   token: {
-    borderRadius: supersetTheme.borderRadius,
+    borderRadius: supersetTheme.borderRadius * 10,
```

Comment:
@geido this is the part that is baffling me. I can change this number (or add a specific one for the Card component, as I've done below) and I'm not seeing the changes in Storybook OR in Superset's UI.